ccvm/ccvm.go

```diff
@@ -185,6 +185,34 @@ func createImages(ctx context.Context, wkld *workload, ws *workspace, args *type
 	return nil
 }
 
+func outputBootingMessage(args *types.CreateArgs, wkld *workload, ws *workspace,
+	resultCh chan interface{}) {
+	spec := &wkld.spec.VM
+
+	resultCh <- types.CreateResult{
+		Line: fmt.Sprintf("Booting VM with %d MiB RAM and %d cpus\n", spec.MemMiB, spec.CPUs),
+	}
+
+	if !args.Debug {
+		return
+	}
+
+	sshPort, err := spec.SSHPort()
+	if err != nil {
+		return
+	}
+
+	resultCh <- types.CreateResult{
+		Line: "To connect to the instance during its creation type\n\n",
+	}
+
+	fstr := "\tssh -q -F /dev/null -o UserKnownHostsFile=/dev/null " +
+		"-o StrictHostKeyChecking=no -o IdentitiesOnly=yes -i %s %s -p %d\n"
```

Comment:
Is it worth refactoring this out so it is common for status/connect/debug? If we need to change it in the future one could easily be missed.

Reply:
I didn't separate it out as the connect and status are in a different package ( and process ).    To separate it out I'd need to create a new package, unless I stick it in types.   I decided a little copying was okay in this case.